src/vm/jitinterface.cpp

```diff
@@ -538,6 +538,15 @@ CEEInfo::ConvToJitSig(
         ULONG data;
         IfFailThrow(sig.GetCallingConvInfo(&data));
         sigRet->callConv = (CorInfoCallConv) data;
+
+#if defined(FEATURE_CORECLR)
+        if (sigRet->callConv & IMAGE_CEE_CS_CALLCONV_VARARG)
```

Comment:
Yes, you are right. It needs to be: `if (isCallConv(sigRet->callConv, IMAGE_CEE_CS_CALLCONV_VARARG))`